### DISCUSSION TITLE
Import data - EBSD

### DISCUSSION THREAD

**Discussion by ArianeNeves** [mtex-toolbox/mtex#1522] [Ask Anything]
Hi!

When I imported a EBSD data, the MTEX identified many notIndexed points. I checked the .ang file and didn't find any errors.
The others maps were imported correctly. 

![image](https://user-images.githubusercontent.com/81368454/183499874-3d05b814-7861-4a6a-ada8-1cabf25a870d.png)

Any tips to fix this error?

**Comment by kilir** [comment]
Hi,
so actually the map should not contain so many non-indexed points? Can share the first 1000s lines of the file (up to some point where there is some data after the header)?

Also, what does `ebsd = EBSD,load()` produce?

Cheers,
Rüdiger

**Comment by ArianeNeves** [comment]
Hi Rüdiger!

The map did not contain so many non-indexed points. 

ebsd = EBSD (show methods, plot)
 
 Phase   Orientations       Mineral         Color  Symmetry  Crystal reference frame
     0  3019096 (95%)    notIndexed                                                 
     1  161428 (5.1%)  Iron (Alpha)  LightSkyBlue       432                         
     2    3195 (0.1%)  Iron (Gamma)  DarkSeaGreen       432                         
 
 Properties: ci, fit, iq, sem_signal, unknown1, unknown2, unknown3, unknown4, x, y, oldId
 Scan unit : um

Plot:

![MTEX](https://user-images.githubusercontent.com/81368454/183635624-debc876f-1444-4811-9e05-64bb57649c0b.jpg)

File:

[MTEX2.txt](https://github.com/mtex-toolbox/mtex/files/9289915/MTEX2.txt)

Thanks, Rüdiger!
